src/services/ws.js

```diff
@@ -91,17 +101,32 @@ export function close() {
 
 /**
  * @param {import("ws").WebSocket} webSocket
- * @param {string} jsonWebToken
+ * @param {Object} credentials
+ * @param {string} [credentials.channelUUID]
+ * @param {string} credentials.jwt
```

Comment:
`credentials` has same shape as `FirstWebSocketPayload`, probably could reconcile the typedef for DRY?